ARS/src/main/java/com/CSC450/ars/controller/HomeController.java

```diff
@@ -51,6 +58,7 @@ public String viewPages(Model model) throws SQLException {
 	
 	@RequestMapping(value="save_page", method=RequestMethod.POST)
 	public String savePage(Model model, @ModelAttribute("page") Page page, BindingResult result) throws SQLException {
+		page.setKeywords(keywordDao.getAll());
```

Comment:
I understand that this method and the following one are only there to test the many to many relationship, but I don't think they will be needed in the final software since all of the objects are created by updating from the Proxy Server.